wdn/templates_4.1/less/layouts/bands.less

```diff
@@ -1,6 +1,14 @@
-// implement the same content padding
-#maincontent > * {
-	.inner-wrapper-margin();
+#maincontent {
+
+    // implement the same content padding
+    > * {
+	    .inner-wrapper-margin();
+    }
+
+    // Add margin to last element (if not a .wdn-band) to add vertical space between main content and footer
+    > *:last-child:not(.wdn-band) {
```

Comment:
This probably could have been done without using a separate nesting level by way of `&`, but it's okay.